private/buf/cmd/buf/command/generate/generate.go

```diff
@@ -310,10 +311,10 @@ func run(
 		container,
 		ref,
 		flags.Config,
-		flags.Paths,        // we filter on files
-		flags.ExcludePaths, // we exclude these paths
-		false,              // input files must exist
-		false,              // we must include source info for generation
+		flags.Paths,                             // we filter on files
+		flags.ExcludePaths,                      // we exclude these paths
+		false,                                   // input files must exist
+		bufimagebuild.WithCanonicalByteOutput(), // include source info, match protoc byte-for-byte
```

Comment:
Regardless of the answers to the above, why do we need to do this for `buf generate`?